pkg/utils/emails/builders.go

```diff
@@ -20,36 +19,20 @@ import (
 	"github.com/sendgrid/sendgrid-go/helpers/mail"
 )
 
-// Email templates must be provided in this directory and are loaded at compile time
-// using go:embed.
-const templatesDir = "templates"
-
 var (
-	//go:embed templates/*.html templates/*.txt
+	//go:embed templates/*.html templates/*.txt templates/*.css templates/partials/*html
 	files     embed.FS
-	templates map[string]*template.Template
+	templates *template.Template
+
+	patterns = []string{
+		"templates/*.html", "templates/*.txt", "templates/*.css",
+		"templates/partials/*.html",
+	}
 )
```

Comment:
In order to use multiple templates to render a single email, we need to have the whole template system loaded.